_posts/About.md

```diff
@@ -0,0 +1,2 @@
+This is an About file. 
```

Comment:
@sarthak240 
Sorry but I don't understand this change.

Why an `About` file is required at all? DO the content of the file is up to the mark?

I think we can improve it. What do u think?